huggingface/tensorflow/inference/docker/2.4/py3/cu110/Dockerfile.gpu

```diff
@@ -12,7 +12,7 @@ ARG PYTHON=python3
 ARG PYTHON_VERSION=3.7.10
 ARG OPEN_MPI_VERSION=4.0.1
 # HF ARGS
-ARG TF_URL=https://aws-tensorflow-binaries.s3-us-west-2.amazonaws.com/tensorflow/r2.4_aws/20210127-150238/gpu/py37/cu110/tensorflow_gpu-2.4.1-cp37-cp37m-manylinux2010_x86_64.whl
+ARG TF_URL=https://aws-tensorflow-binaries.s3-us-west-2.amazonaws.com/tensorflow/r2.4_aws/20210127-150238/gpu/py37/cu110/tensorflow_gpu-2.4.3-cp37-cp37m-manylinux2010_x86_64.whl
```

Comment:
I don't think these links will be valid - do you just generally use the one from the tensorflow DLC? i.e. here? https://github.com/aws/deep-learning-containers/blob/master/tensorflow/training/docker/2.4/py3/cu110/Dockerfile.gpu#L37